scripts/bulk-password-update-export.js

```diff
@@ -106,30 +127,49 @@ const loadUsers = async () => {
   }
 };
 
-const changeUserPass = async (user, postOptions) => {
-  const newPass = await generatePassword();
-  postOptions.body = {
-    password: newPass
-  };
-  postOptions.uri = `${options.uri}/${user}`;
-  try {
-    await rpn.post(postOptions);
-    console.log('SUCCESS', user, newPass);
-  } catch (e) {
-    console.log('ERROR', user, e.message);
+
+// Thanks https://stackoverflow.com/a/40672956
+String.prototype.splitCSV = function() {
+  let matches = this.match(/(\s*"[^"]+"\s*|\s*[^,]+|,)(?=,|$)/g);
+  for (let n = 0; n < matches.length; ++n) {
+    matches[n] = matches[n].trim();
+    if (matches[n] === ',') matches[n] = '';
+  }
+  return matches;
+}
```

Comment:
CSVs are notoriously hard to parse so let's not write a bespoke parser when we have the whole of npm at our fingertips. In fact, the `load_messages` script already uses a csv parser so just use that one!

Reply:
[Papaparse](https://www.papaparse.com/docs) is good for this too.

Reply:
@garethbowen  FTW in 63025e3!